tools/walletextension/main/testnet.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Is it possible that we put this in a gh secret and then pull it out, https://tinyurl.com/yc78scsm

Reply:
added that, but I think we will need to change it also for L2 deployment. The same testnet.pem file is also there.